### PR TITLE
cleanup of ase after aks migration

### PR DESCRIPTION
**Pull request by naikajah**
Once the ASE to AKS migration is completed. This PR should be marked ready for review and then merged updating the value of
enable_ase to be false for the environment that needs ASE's to be deleted .
e.g.

>> aat.tfvars
>> enable_ase = false

### CODE REVIEW

File: infrastructure/test_config.tf [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (jackmaloney) [inline infrastructure/test_config.tf]: Is this file needed?

Comment (naikajah) [reply infrastructure/test_config.tf]: No idea. Probably not. I find this file in almost all the repositories. So just left it l;ike that. But if not needed we can remove it .

Comment (timja) [reply infrastructure/test_config.tf]: not needed
